crescent/commands/groups.py

```diff
@@ -64,6 +81,25 @@ def child(self, includable: Includable[AppCommandMeta]) -> Includable[AppCommand
 
 @dataclass
 class SubGroup:
+    """
+    A command subgroup. A command subgroup is a group that is under a top level group.
+
+    ### Example
+    ```python
+    import crescent
+
+    utils_group = crescent.Group("utils")
+    time_utils_group = utils_group.sub_group("time")
+
+    # This command will appear under the `utils` group in discord.
+    @client.include
+    @time_utils_group.child
+    @crescent.command
+    async def latency(ctx: crescent.Context):
+        await ctx.respond(f"The latency is {bot.heartbeat_latency * 1000}ms")
+    ```
+    """
+
```

Comment:
I notice an inconsistent usage of a newliine after the end of docstrings before code - here there's a newline, but in most of the other cases there's no newline.